connexion/operations/swagger2.py

```diff
@@ -0,0 +1,291 @@
+import logging
```

Comment:
Curious, was this just stripped out from an older file into its own file to ensure backwards compatibility?

Reply:
Yup, the swagger2.py file is for backwards compatibility with the swagger2 spec. The idea with this PR is to create an Abstract Base Class (abc), which is essentially the interface that describe what an "Operation" is. Previously we only had one implementation of this interface (swagger 2 Operation), so it wasn't well defined. Now, after this refactor, the OpenAPIOperation can implement the exact same interface as a Swagger2Operation. Together they encapsulate the differences between the Swagger2 spec and the OpenAPI3 spec. Hope that makes sense.